Cyberduck/Profiles.md

```diff
@@ -11,6 +11,54 @@ These files ([XML Property List Format](http://en.wikipedia.org/wiki/Property_li
 
 [Contact us](mailto:support@cyberduck.io) if you are a service provider and need assistance in setting this up.
 
+
+# Preferences → Profiles
+
+```{note}
+Prelimitary documentation for Cyberduck version 8
+``` 
+
+Select connection profiles to be installed. Either scroll through the list or use the search function to look for a specific profile. The connection profiles will be installed after ticking the corresponding checkboxes. Installed protocols are displayed in the _Protocol_ section of the bookmark window. To disable the connection profile simply untick the checkbox. The profile will be disabled after closing the application.
```

Comment:
> Installed protocols are displayed in the _Protocol_ section of the bookmark window.

Misleadingly formulated, since there is no field in the bookmark configuration window that is named _Protocol_.